plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_launcher_util.py

```diff
@@ -1,28 +1,27 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 import logging
+import os
 from contextlib import contextmanager
 from subprocess import PIPE, Popen
-from typing import Any, Dict, Generator, Tuple
+from typing import Any, Dict, Generator, List, Tuple
 
 import ray
 from hydra.core.config_search_path import ConfigSearchPath
 from hydra.core.hydra_config import HydraConfig
 from hydra.core.singleton import Singleton
-from hydra.core.utils import (
-    JobReturn,
-    run_job,
-    setup_globals,
-)
+from hydra.core.utils import JobReturn, run_job, setup_globals
 from hydra.plugins.search_path_plugin import SearchPathPlugin
 from hydra.types import TaskFunction
-from omegaconf import DictConfig
+from omegaconf import DictConfig, OmegaConf
 from ray._raylet import ObjectID
 
 log = logging.getLogger(__name__)
 
 JOB_SPEC_PICKLE = "job_spec.pkl"
 JOB_RETURN_PICKLE = "returns.pkl"
 
+RAY = "ray-autoscaler"
+
```

Comment:
I don't particularly like polluting the module namespace for things I am not expecting users to use.
Is this being used from outside ray_launcher_utils?